tests/e2e/e2e_setup_test.go

```diff
@@ -26,25 +26,37 @@ import (
 )
 
 const (
-	osmoBalance  = 200000000000
-	osmoDenom    = "uosmo"
-	stakeBalance = 110000000000
-	stakeAmount  = 100000000000
-	stakeDenom   = "stake"
-	minGasPrice  = "0.00001"
+	// common
+	osmoDenom   = "uosmo"
+	stakeDenom  = "stake"
+	minGasPrice = "0.00001"
+	// chainA
+	chainAName    = "osmo-test-a"
+	osmoBalanceA  = 200000000000
+	stakeBalanceA = 110000000000
+	stakeAmountA  = 100000000000
+	// chainB
+	chainBName    = "osmo-test-b"
+	osmoBalanceB  = 500000000000
+	stakeBalanceB = 440000000000
+	stakeAmountB  = 400000000000
 )
 
 var (
-	initBalanceStr    = fmt.Sprintf("%d%s,%d%s", osmoBalance, osmoDenom, stakeBalance, stakeDenom)
-	stakeAmountInt, _ = sdk.NewIntFromString(fmt.Sprintf("%d", stakeAmount))
-	stakeAmountCoin   = sdk.NewCoin(stakeDenom, stakeAmountInt)
+	initBalanceStrA    = fmt.Sprintf("%d%s,%d%s", osmoBalanceA, osmoDenom, stakeBalanceA, stakeDenom)
+	initBalanceStrB    = fmt.Sprintf("%d%s,%d%s", osmoBalanceB, osmoDenom, stakeBalanceB, stakeDenom)
+	stakeAmountIntA, _ = sdk.NewIntFromString(fmt.Sprintf("%d", stakeAmountA))
```

Comment:
You can avoid the costly `Sprintf` call and just do `NewInt` which should take a native `int64`